src/ZMQ.jl

```diff
@@ -4,7 +4,7 @@ VERSION >= v"0.4.0-dev+6521" && __precompile__(true)
 
 module ZMQ
 using Compat
-import Compat.String
+import Compat: String, unsafe_string
```

Comment:
NVM, the `@deprecate` seems fine since the `Compat.unsafe_string` is not an alias. So only the export one.